core/trino-main/src/main/java/io/trino/operator/scalar/MathFunctions.java

```diff
@@ -1418,4 +1419,30 @@ private static Double mapL2Norm(Block map)
 
         return Math.sqrt(norm);
     }
+
+    @Description("ChiSquared cdf given the df parameter and value")
+    @ScalarFunction("chi_squared_cdf")
+    @SqlType(StandardTypes.DOUBLE)
+    public static double chiSquaredCdf(
+            @SqlType(StandardTypes.DOUBLE) double df,
+            @SqlType(StandardTypes.DOUBLE) double value)
+    {
+        checkCondition(value >= 0, INVALID_FUNCTION_ARGUMENT, "value must non-negative");
+        checkCondition(df > 0, INVALID_FUNCTION_ARGUMENT, "df must be greater than 0");
+        ChiSquaredDistribution distribution = new ChiSquaredDistribution(null, df, ChiSquaredDistribution.DEFAULT_INVERSE_ABSOLUTE_ACCURACY);
```

Comment:
Any reason on why we pass `RandomGenerator` as null ?

Reply:
Do we need to customize the absolute accuracy ?

Reply:
[Here](http://home.apache.org/~luc/commons-math-3.6-RC2-site/apidocs/org/apache/commons/math3/distribution/ChiSquaredDistribution.html) it shows the constructor having the potential for a RandomGenerator, but I don't believe it to be necessary on this baseline code. I have confirmed this code works on Presto, therefore I kept it. 

Same with customizing absolute accuracy. We do not need this currently. 

In the future, these could be changed.